vscode/webviews/components/shadcn/ui/toolbar.tsx

```diff
@@ -92,36 +92,33 @@ export const ToolbarButton = forwardRef<HTMLButtonElement, ToolbarButtonProps>(
 )
 ToolbarButton.displayName = 'ToolbarButton'
 
-export const ToolbarPopoverItem: FunctionComponent<
+export const ToolbarPopoverItem = forwardRef<
+    { open: () => void },
```

Comment:
```suggestion
    { open: () => void, close: () => void },
```

I think this was why Escape didn't work for me 🤔  not sure why it was working for you?

Reply:
I wonder if it's a global command command that's taking over?